mobile/src/layout/tab-navigation-options-factory.tsx

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import { NavigationTabScreenOptions } from 'react-navigation';
 import Icon from 'react-native-vector-icons/Ionicons';
+import { Image } from 'react-native';
+import { ImageURISource } from 'react-native';
 
 
 export class TabNavigationOptionsFactory {
-    public create(label: string, icon: string, unfocusedIcon: string): NavigationTabScreenOptions {
+    public create(label: string, focusedPath: ImageURISource, unfocusedPath: ImageURISource): NavigationTabScreenOptions {
```

Comment:
is it really an instance of `ImageURISource`? What does this thing look like in the console? I do not know the answer, just asking ;)

Reply:
It is type for image source(like string, number and etc).